bindings/python/iota_sdk/types/output_id.py

```diff
@@ -2,64 +2,66 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from dataclasses import dataclass
-
-from iota_sdk.types.common import HexStr, json
+from iota_sdk.types.common import json
 from iota_sdk.types.output import Output
 from iota_sdk.types.transaction_id import TransactionId
 
 
-class OutputId(dict):
+class OutputId(str):
     """Represents an output ID.
 
     Attributes:
         output_id: The unique id of an output.
-        transaction_id: The transaction id associated with the output.
-        output_index: The index of the output within a transaction.
-
     """
 
-    def __init__(self, transaction_id: TransactionId, output_index: int):
+    def __new__(cls, output_id: str):
         """Initialize OutputId
         """
-        if len(transaction_id) != 74:
+        if len(output_id) != 78:
             raise ValueError(
-                'transaction_id length must be 74 characters with 0x prefix')
-        if not transaction_id.startswith('0x'):
-            raise ValueError('transaction_id must start with 0x')
-        # Validate that it has only valid hex characters
-        int(transaction_id[2:], 16)
-        output_index_hex = (output_index).to_bytes(2, "little").hex()
-        self.output_id = transaction_id + output_index_hex
-        self.transaction_id = transaction_id
-        self.output_index = output_index
+                'output_id length must be 78 characters with 0x prefix')
+        if not output_id.startswith('0x'):
+            raise ValueError('output_id must start with 0x')
+        instance = super().__new__(cls, output_id)
```

Comment:
What is `super` here? `str`?

Reply:
Yes

Reply:
So apart from length and 0x, we don't really check that it's hexadecimal ?

Reply:
No, added